packages/api/src/api.ts

```diff
@@ -112,12 +114,19 @@ const RecommendationsApiMiddleware = async function (
   }
 }
 
-const router = express.Router()
+const createRouter = (config?: CCFConfig) => {
+  setConfig(config)
 
-router.get('/footprint', FootprintApiMiddleware)
-router.get('/regions/emissions-factors', EmissionsApiMiddleware)
-router.get('/recommendations', RecommendationsApiMiddleware)
-router.get('/healthz', (req: express.Request, res: express.Response) => {
-  res.status(200).send('OK')
-})
-export default router
+  const router = express.Router()
+
+  router.get('/footprint', FootprintApiMiddleware)
+  router.get('/regions/emissions-factors', EmissionsApiMiddleware)
+  router.get('/recommendations', RecommendationsApiMiddleware)
+  router.get('/healthz', (req: express.Request, res: express.Response) => {
+    res.status(200).send('OK')
+  })
+
+  return router
+}
+
+export default createRouter
```

Comment:
Let's make this a named export on line 117, rather than the default export. This is to ensure naming consistency when importing and using the function. E.g. in the `server.ts` and test files.